.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,23 @@
 ---
 name: Bug report
-about: Create a report to help us improve
-title: "⚠️"
+about: Create a report to help us improve Editor.js
+title: "[Bug]"
 labels: bug
 assignees: ''
 
 ---
 
-**Describe the bug**
-A clear and concise description of what the bug is.
+Describe a problem.
```

Comment:
I think it is better to use a "bug" instead of a "problem" because "problem" is more abstract (it might define a task/goal/execise)